skimage/filters/thresholding.py

```diff
@@ -881,3 +881,42 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
         r = 0.5 * (imax - imin)
     m, s = _mean_std(image, window_size)
     return m * (1 + k * ((s / r) - 1))
+
+
+def apply_hysteresis_threshold(image, low, high):
```

Comment:
>Maybe I'll add a call to np.clip and add the new types to the docstring.

@jni Great, thanks! Take your time :wink: . The enhancement will be non-desctructive, in any case, so I'm merging.